app/components/publish/providers/study_sites/explainer_component.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module Publish
+  module Providers
+    module StudySites
+      class ExplainerComponent < ApplicationComponent
```

Comment:
🤔 why have a component for this with nothing in it? Why not just render the html directly in the view?

Reply:
I choose a component for both of these to enable easy testing. The alternative would be to spin up a system/feature test to visit the page, which felt like overkill.